docs/source/examples/spot-jobs.rst

```diff
@@ -166,7 +166,7 @@ We also set :code:`--run_name` to :code:`$SKYPILOT_RUN_ID` so that the loggings
 to the same run in Weights & Biases.
 
 .. note::
-  The environment variable :code:`$SKYPILOT_RUN_ID` can be used to identify the same job, i.e., it is kept identical across all
+  The environment variable :code:`$SKYPILOT_RUN_ID` (example: "sky-2022-10-06-05-17-09-750781_spot_id-22") can be used to identify the same job, i.e., it is kept identical across all
```

Comment:
A quick question: What does 750781 mean here? Can we make the example easier to understand?

Reply:
Got it. Yeah, I don't have a better idea now. Let's keep it.